ci/common/suite.sh

```diff
@@ -39,10 +39,11 @@ enter_suite() {
 
 exit_suite() {
   set +x
-  travis_fold end "${NVIM_TEST_CURRENT_SUITE}"
   if test $FAILED -ne 0 ; then
     echo "Suite ${NVIM_TEST_CURRENT_SUITE} failed, summary:"
     echo "${FAIL_SUMMARY}"
+  else
+    travis_fold end "${NVIM_TEST_CURRENT_SUITE}"
```

Comment:
See https://travis-ci.org/neovim/neovim/builds/553115834 - looks good.
Happy to pull this out of here.

Reply:
Pushed in d42bbf363.